specification/schema/tile.schema.json

```diff
@@ -65,33 +65,25 @@
             ]
         },
         "content": {
-            "description": "Metadata about the tile's content and a link to the content. When this is omitted the tile is just used for culling.",
+            "description": "Metadata about the tile's content and a link to the content. When this is omitted the tile is just used for culling. When this is defined, then `contents` must be undefined.",
             "$ref": "content.schema.json",
             "deprecated": true
```

Comment:
I think I added `"deprecated": true` here originally in https://github.com/CesiumGS/3d-tiles/pull/634 but it should be removed. I wouldn't expect `content` to actually be removed in the future.